packages/lib/src/interactions/utils.ts

```diff
@@ -20,8 +21,14 @@ export function boundPointToFOV(
 export function getRatioRespectingRectangle(
   startPoint: Vector2,
   endPoint: Vector2,
-  ratio: number
-) {
+  xVisibleDomain: Domain,
+  yVisibleDomain: Domain
```

Comment:
The signature was changed to match the signature of the new `getDistinctPoints` and make the functions interchangeable.